components/styled/icon/src/molecules/emotion-icon.tsx

```diff
@@ -0,0 +1,25 @@
+import { EmotionIconType, IconStatus, IconDirection } from '@ltht-react/styles'
+import { FC, HTMLAttributes } from 'react'
+import ChevronDoubleIcon from '../atoms/emotion-icons/chevron-double-icon'
+
+const EmotionIcon: FC<EmotionIconProps> = ({ type, size, status, direction, counterValue, ...rest }) => {
```

Comment:
We shouldn't need to deconstruct `counterValue` since we're not using it - I guess it's left over copypasta from the original IconProps interface where it was used for the counter icon